tululbot/utils.py

```diff
@@ -120,7 +120,20 @@ def refresh_cache(self):
 
 def lookup_slang(word):
     not_found_word = 'Gak nemu cuy'
-    return lookup_urbandictionary(word) or lookup_kamusslang(word) or not_found_word
+    return lookup_slang_sources(word) or not_found_word
+
+
+def lookup_slang_sources(word):
+    urbandic_def = lookup_urbandictionary(word)
```

Comment:
- [ ] please be consistent on how you abrreviate `urbandictionary`: `urban`, `urbandic`, (or `urbandict`)?